apiary.apib

```diff
@@ -374,6 +374,10 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- The task assignee is now nullable: you can create / update unassigned tasks by sending null as assignee. The tasks.list and tasks.info endpoints also return null as assignee in case the task is unassigned.
+
+- On API versions older than 2019-10-09, the assignee is returned as { type: user, id: null } in case the task is unassigned.
```

Comment:
To me `{ type: user, id: null }` looks a bit strange. I would think that sending back `null` as the id is as breaking as simply sending back `null` as the assignee.

Looks like it's very difficult to keep this working in a backward-compatible way.

Or we need to send back the creator as the assignee and tell our users to upgrade to a newer API version if they want to take advantage of the unassigned tasks.

WDYT?

Reply:
But sending the creator of the task as the assignee is literally wrong data. Also; wether you get `null` or a `UUID` doesn't really matter. What if the user has been deleted? Integrations should already be expecting UUID's to potentially not match a user. But they might not be expecting the reference to be gone all together.

The only problem I see with this is our validation failing for non-valid UUID's.